### PR TITLE
Support polygon closing styling

### PR DESCRIPTION
**Pull request by xintongxia**
demo

### CODE REVIEW

**Comment by ibgreen** [comment]
Worth a screenshot?

**Comment by georgios-uber** [comment]
Screenshot please? And some unit tests would be nice **Change from base Build 1184: -0.3%**

File: modules/react-map-gl-draw/src/editor.js [hunk 1]
```diff
@@ -618,18 +618,25 @@ export default class Editor extends PureComponent<EditorProps, EditorState> {
...
       ? renderState
-      : index === draggingVertexIndex ||
-        (selectedFeature && selectedFeature.renderType === RENDER_TYPE.POINT)
-        ? RENDER_STATE.SELECTED
-        : index === hoveredVertexIndex
-          ? RENDER_STATE.HOVERED
-          : index === -1
-            ? RENDER_STATE.UNCOMMITTED
-            : RENDER_STATE.INACTIVE;
+      : isClosing
```
Comment (georgios-uber) [inline modules/react-map-gl-draw/src/editor.js]: switch?

Comment (georgios-uber) [reply modules/react-map-gl-draw/src/editor.js]: or better way to be readable

**Comment by xintongxia** [comment]
> Screenshot please? And some unit tests would be nice **Change from base Build 1184: -0.3%**

I am working on refactoring to extend from `@nebula.gl/edit-mode`, then I'll start adding tests from there.

**Pull request merged**